ivy.el

```diff
@@ -3227,12 +3227,20 @@ Otherwise, ~/ will move home."
 
 (defvar ivy--exhibit-timer nil)
 
+(let (last-pos)
+  (defun ivy-input-changed? ()
+    (let* ((pos (line-end-position))
+           (changed (not (eq pos last-pos))))
```

Comment:
It's better to use `eql` for numeric comparisons, as it can handle both fixnums and bignums. Using `eq` (which only works for fixnums) for this is considered a bit of a hack / passé.